web/packages/extension/webpack.config.js

```diff
@@ -42,12 +42,31 @@ module.exports = (env, argv) => {
                         to: "..",
                         transform(content) {
                             const manifest = JSON.parse(content.toString());
-                            const { version } = require("./package.json");
-                            Object.assign(manifest, { version });
+
+                            const packageVersion =
+                                process.env.npm_package_version;
+                            const versionChannel =
+                                process.env.CFG_RELEASE_CHANNEL || "nightly";
+                            const buildDate = new Date()
+                                .toISOString()
+                                .substr(0, 10);
+                            // The extension marketplaces require the version to monotonically increase,
+                            // so append the build date onto the end of the manifest version.
+                            const version = `${packageVersion}.${buildDate.replace(
```

Comment:
`buildDate.replaceAll("-", "")` can be used, though `replaceAll` is available only since Node.js 15.

Reply:
CI is running node 14 currently, so let's keep this for now.